### ISSUE TITLE
Unable to build an extension of inbound-agent:*ltsc2019

### ISSUE THREAD

**Issue by Vlatombe** [jenkinsci/docker-agent#815]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Paste the output here
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

N/A

### Reproduction steps

N/A

### Expected Results

The image build succeeds

### Actual Results

Getting the following error when extending the image
```
#8 [2/2] COPY agent.jar C:/ProgramData/Jenkins/agent.jar
--
  | 110707 | 01:09:48 PM | #8 ERROR: mount callback failed on /tmp/containerd-mount2303027507: link /tmp/containerd-mount2303027507/Windows/INF/basicrender.inf /tmp/containerd-mount2303027507/Windows/System32/DriverStore/FileRepository/basicrender.inf_amd64_efdc64af60c69a6d/basicrender.inf: no such file or directory
```

### Anything else?

I'm hitting https://github.com/microsoft/Windows-Containers/issues/493 when trying to extend the inbound-agent image built for ltsc2019 (using docker buildx on linux host).

This was not happening with previous versions.
From the issue referenced above, a workaround (until Microsoft fixes the upstream issue) is to pin the base windows image to the April release

**Comment by jglick** [comment]
Resolved by #816 (at least for now)?

**Comment by dduportal** [comment]
> Resolved by #816 (at least for now)?

Absolutely! I was waiting for the images to be fully released before closing, which is done:

- Images are available: https://hub.docker.com/r/jenkins/inbound-agent/tags?page=&page_size=&ordering=&name=3248.v65ecb_254c298-4
- https://github.com/jenkinsci/docker-agent/releases/tag/3248.v65ecb_254c298-4 
Thanks folks!

**Comment by lemeurherve** [comment]
I tried to reproduce the error from jenkins/inbound-agent:3248.v65ecb_254c298-3 (the build before the fix here), but it passed:
```
FROM jenkins/inbound-agent:3248.v65ecb_254c298-2 AS build-ko
COPY README.md README.md
```

@Vlatombe do you have an example of a Dockerfile reproducing this error?

I'd like to be able to confirm that the last image published today effectively fix the issue to update the pinned image version.

From https://github.com/microsoft/Windows-Containers/issues/493#issuecomment-2219090926:

> The updated images released as part of the July 2024 security update today include the fix for this issue.
